src/unit_tests.zig

```diff
@@ -42,4 +42,6 @@ test {
     _ = @import("vsr/superblock_quorums.zig");
 
     _ = @import("aof.zig");
+
+    _ = @import("Shell.zig");
```

Comment:
(I lied when I said one more)
```suggestion
    _ = @import("shell.zig");
```